app/scripts/directives/blockEditor.js

```diff
@@ -116,12 +116,17 @@ angular.module('confRegistrationWebApp')
           if (angular.isUndefined($scope.block.content)) {
             $scope.block.content = { 'choices': [] };
           } else if(angular.isUndefined($scope.block.content.choices)){
-            $scope.block.content.choices = [];
+              $scope.block.content.choices = [];
+          }
+          if (_.some($scope.block.content.choices, ['value', newOption])){
+            modalMessage.error('Error: Cannot use the same Multiple Choice Answer more than once.');
```

Comment:
It looks like this code applies to multiple choice, checkbox, and dropdown questions. Can you make the message more generic? Maybe `You may not use the same answer more than once for this question. Each answer needs to be unique.`. Idk. I think I tend to make things more verbose haha.

Also `Error:` seems duplicate:
<img width="611" alt="screen shot 2017-11-29 at 7 55 49 pm" src="https://user-images.githubusercontent.com/756501/33412306-5e0e4672-d53f-11e7-9a23-37b62d2da84c.png">

If you end up doing inline validation when editing existing fields, it might make sense to do inline validation here too.

Reply:
I modified the error to use your more descriptive message.

Reply:
👍